src/Miningcore/Extensions/DictionaryExtensions.cs

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright 2017 Coin Foundry (coinfoundry.org)
+Authors: Oliver Weichhold (oliver@weichhold.com)
```

Comment:
this is a wierd one.  we're adding a file (with OliverW's Copyright)  that wasn't in Oliver's repo?